pkg/data/utils.go

```diff
@@ -8,10 +8,10 @@ import (
 	"io"
 	"os"
 	"strings"
-
-	"github.com/arvyshka/blazer/internals"
 )
 
+const MemUnit = 1024
```

Comment:
for consistency across the project, we can follow all constants as CAPS policy.

Reply:
@setraj : our linters take care of it, to start it you need to run `make lint`. We need to make it part of our CI/CD flow. I tried to integrate github's CI/CD but it is not working as expected.. have you used github's CI/CD if not what do you suggest ?